doc/pages/expansions.asciidoc

```diff
@@ -252,6 +252,14 @@ The following expansions are supported (with required context _in italics_):
     _in window scope_ +
     quoted list of the contents of all selections
 
+*%val{selection_length}*::
+    _in window scope_ +
+    length (in codepoints) of the main selection
+
+*%val{selections_length}*::
+    _in window scope_ +
+    comma-separated list of the lengths (in codepoints) of the selections
+
```

Comment:
Currently, the expansions are listed in approximately alphabetical order, so you probably want to put these below `selection_desc` and `selections_desc`.

Reply:
Good points